exercises/transpose/transpose_test.go

```diff
@@ -0,0 +1,41 @@
+package transpose
+
+import (
+	"fmt"
+	"math"
+	"testing"
+)
+
+// For a more visual representaion of the test data than is provided by
+// cases_test.go have a look at the original json data here:
+// https://github.com/exercism/x-common/blob/master/exercises/transpose/canonical-data.json
+
+func TestTranspose(t *testing.T) {
+	for _, test := range testCases {
+		actual := Transpose(test.input)
+		if fmt.Sprintf("%s", actual) != fmt.Sprintf("%s", test.expected) {
```

Comment:
any risk of a false positive from this? For example:

```go
package main

import (
	"fmt"
)

func main() {
	a := []string{"a b", "c"}
	b := []string{"a", "b c"}
	fmt.Printf("%s and %s are equal? %t", a, b, fmt.Sprintf("%s", a) == fmt.Sprintf("%s", b))
}
```

https://play.golang.org/p/uvtXMhdJ8W